nemo_skills/prompt/config/generic/mmlu_pro.yaml

```diff
@@ -0,0 +1,20 @@
+few_shot_examples:
```

Comment:
Also if that's not too much work, can you please change the regular mmlu to use the same "options" style, so that we can reuse the config for that as well? And can we also reuse your new mmlu_eval there?

Reply:
I don't know which MMLU format is standard at the moment, I believe there are quite a lot of them.
I'm thinking to research on MMLU format later

Reply:
ok, let's keep it for later